engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -435,6 +437,15 @@ public static String isValidImportString(Logger log, String importString) {
         }
     }
 
+    private static boolean visibleToClassGraph(String packageImport) {
+        try (ScanResult scanResult = new ClassGraph().enableAllInfo().acceptPackages(packageImport).scan()) {
```

Comment:
I think we can limit the scan features to `ClassGraph().enableClassInfo()`; I don't think we necessarily want, or need, the other features (field info, method info, annotation info, private classes, etc).